src/store/configureStore.dev.ts

```diff
@@ -31,40 +17,30 @@ import {GlobalState} from 'types/store';
  * Configures and constructs the redux store. Accepts the following parameters:
  * preloadedState - Any preloaded state to be applied to the store after it is initially configured.
  * appReducer - An object containing any app-specific reducer functions that the client needs.
- * userOfflineConfig - Any additional configuration data to be passed into redux-offline aside from the default values.
+ * persistConfig - Any additional configuration data to be passed into redux-persist aside from the default values.
  * getAppReducer - A function that returns the appReducer as defined above. Only used in development to enable hot reloading.
  * clientOptions - An object containing additional options used when configuring the redux store. The following options are available:
  *     additionalMiddleware - func | array - Allows for single or multiple additional middleware functions to be passed in from the client side.
  *     enableThunk - bool - default = true - If true, include the thunk middleware automatically. If false, thunk must be provided as part of additionalMiddleware.
  */
-export default function configureServiceStore(preloadedState: any, appReducer: any, userOfflineConfig: any, getAppReducer: any, clientOptions: any) {
-    const baseOfflineConfig = Object.assign({}, defaultOfflineConfig, offlineConfig, userOfflineConfig);
+export default function configureServiceStore(preloadedState: any, appReducer: any, persistConfig: any, getAppReducer: any, clientOptions: any): Store {
```

Comment:
Strictly speaking, this is a breaking change for anyone using mattermost-redux to configure their own store. This isn't one of the areas I think we've really implied backwards compatibility though. If we ever do that, we should make this take named arguments in an object instead.

Reply:
Agreed re: breaking change, and that we're not explicitly supporting backwards compatibility here. I'm not really sure we should, either, but open for debate!